test/tests/functional/pbs_accumulate_resc_used.py

```diff
@@ -397,16 +405,20 @@ def test_prologue(self):
         #
         # For string_array type  resource 'stra', it is not accumulated but
         # will be set to last seen value from a mom prologue hook.
-        self.server.expect(JOB, {
+        a = {
             'job_state': 'F',
             'resources_used.foo_f': '0.35',
             'resources_used.foo_i': '35',
             'resources_used.foo_str4': "eight",
             'resources_used.stra': "\"glad,elated\",\"happy\"",
             'resources_used.vmem': '35gb',
-            'resources_used.cput': '00:00:35',
-            'resources_used.ncpus': '3'},
-            extend='x', offset=10, attrop=PTL_AND, id=jid)
+            'resources_used.ncpus': '3'}
+
+        if has_cpuset is 0:
```

Comment:
Say if not  has_cpuset: (here and other places)

Reply:
hahaha.  I originally had that, then the grammar of it bugged me, so I changed it.  I'll change it back.